svwp_to_nature_id_none.json

```diff
@@ -0,0 +1,860 @@
+[
+    {
+        "wp_article": "Voxnan"
+    },
+    {
+        "wp_article": "Österdalälven"
+    },
+    {
+        "wp_article": "Årsta holmar"
```

Comment:
A thing that's tricky is that some of the articles about planned reserves could be outdated and the reserve might have been created. Also, right now Kategori:Planerade naturreservat i Sverige only has six items in it, so it's not a widespread problem.

I have added a line in find_wp_reserve_in_data_file() to only match if the status in the source file is GÄLLANDE, which will cut down on the number of multiple matches a little (when a reserve has two entries, as both BESLUTAT and GÄLLANDE for example).